components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/utils/GitpodLogger.kt

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package io.gitpod.jetbrains.remote.utils
+
+import com.intellij.openapi.diagnostic.logger
+
+class GitpodLogger {
```

Comment:
I would prefer we don't do it, you won't have a class in logs anymore, it looks like:
```
2022-06-30 14:24:17,323 [ 141332]   INFO - #i.g.j.r.u.GitpodLogger - [Gitpod Backend Plugin] Forwarded port 5900 from Supervisor's Terminal 63 to client's port 5900.
```

vs 

```
GitpodProjectManager - gitpod: 'gitpod-pr-10986' project: SDK was auto preconfigured: 11 (/home/gitpod/.sdkman/candidates/java/11.0.15.fx-zulu)
```